ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -155,6 +155,14 @@ Listen <%= addr_port %>
   #
   SetEnv OOD_PUN_STAGE_CMD "<%= @pun_stage_cmd %>"
 
+  <%- if @pun_pre_hook_exports -%>
+  # Environment variables to export to the PUN pre hook.
+  SetEnv OOD_PUN_PRE_HOOK_EXPORTS "<%= @pun_pre_hook_exports %>"
+  <%- if @pun_pre_hook_exports -%>
+  SetEnv OOD_PUN_PRE_HOOK_ROOT_CMD "<%= @pun_pre_hook_root_cmd %>"
+  <%- end -%>
+
+  <%- end -%>
```

Comment:
Might wrap each variable SetEnv in it's own if condition right? otherwise if someone doesn't set exports they won't set pre hook or is that intended?

Reply:
Yea I had the conditions wrong. It should be if the pre_hook write it, then nested within that if there are exports, write them.